src/cc/export/helpers.h

```diff
@@ -90,6 +90,14 @@ struct _name##_table_t { \
 __attribute__((section("maps/perf_array"))) \
 struct _name##_table_t _name
 
+// Identifier for current CPU used in perf_submit and perf_read
+// Prefer use BPF_F_CURRENT_CPU flag, falls back to call helper for older kernel
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 8, 0)
```

Comment:
we got rid of all KERNEL_VERSION() checks, since they're not reliable due to backports.
imo we need a different way to check.
Unfortunately no good suggestions.